modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm

```diff
@@ -55,6 +55,8 @@
 /// The ratio of reagents that get purged while a Hemophage vomits from trying to eat/drink something that their tumor doesn't like.
 #define HEMOPHAGE_VOMIT_PURGE_RATIO 0.95
 
+/// The rate at which blood metabolizes in a Hemophage's stomach subtype.
+#define BLOOD_METABOLIZATION_RATE 0.1 * REAGENTS_METABOLISM
```

Comment:
```suggestion
#define BLOOD_METABOLIZATION_RATE (0.1 * REAGENTS_METABOLISM)
```
good form and to make sure PEMDAS is followed

Reply:
thank you.